debian/control

```diff
@@ -8,6 +8,5 @@ Standards-Version: 3.9.4
 Package: google-cloud-ops-agent
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Conflicts: stackdriver-agent, google-fluentd
```

Comment:
Is this change related?

Reply:
No, thanks for catching that. It's unrelated experimentation that I accidentally left in my environment.